### ISSUE TITLE
Help me get it running 

### ISSUE THREAD

**Issue by FishieMtg** [farag2/Windows-10-Sophia-Script#94]
Im on 20H2 and cant get it to download neither through Github nor rd pary UI programmers 
When i Try to download it it ju says virus detected and yes i tried everything to do that i know of going as far as disabaling all virus checks in vindows defender and the internet security manager or what is called (the thing you go to by typing  inetcpl.cpl in the stert menu)  im new to powershell stuff and im sorry if im bothering anyone but im really interested in this skript/program as ive heared alot of positive things about it 
And if you could help me please try and dumb it down a bit as Im really new to this sort of stuff Thank you in advance

**Comment by farag2** [comment]
Seems you are running Chrome. Will help tomorrow. :)

**Comment by farag2** [comment]
Upload a pic with an error.

**Comment by FishieMtg** [comment]
it doesnt really show an error code or what specifically is wrong it just
says (translated) error  virus
[image: image.png]

Am Mi., 18. Nov. 2020 um 12:03 Uhr schrieb Dmitry Nefedov <
notifications@github.com>:

**Comment by farag2** [comment]
No pic... Can I help you with Discord?

**Comment by FishieMtg** [comment]
sure my discord is FischieMtg#7987

Am Mi., 18. Nov. 2020 um 12:09 Uhr schrieb Dmitry Nefedov <
notifications@github.com>:
